kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/order/CreateExternalOrderCall.java

```diff
@@ -123,13 +135,20 @@ private void runOnMainThread(Runnable runnable) {
 
     interface ExternalOrderCallbacks {
 
+        void onOrderConfirmed(String confirmationJwt);
+
+        void onOrderFailed(String msg);
+    }
+
+    interface ExternalSpendOrderCallbacks extends ExternalOrderCallbacks {
+
         void onTransactionSent(OpenOrder openOrder);
 
         void onTransactionFailed(OpenOrder openOrder, String msg);
+    }
 
-        void onOrderConfirmed(String confirmationJwt);
-
-        void onOrderFailed(String msg);
+    interface ExternalEarnOrderCallbacks extends ExternalOrderCallbacks {
 
+        void onOrderCreated(OpenOrder openOrder);
```

Comment:
onOrderCreated can also be part of ExternalOrderCallbacks - the order is also created for spend transaction right?